src/Utils/OrderTimeHelper.php

```diff
@@ -37,24 +37,34 @@ public function __construct(
     {
     }
 
-    private function filterChoices(OrderInterface $cart, array $choices): array
+    private function filterChoices(OrderInterface $cart, array $choices, $fulfillmentMethod): array
     {
         $choicesLogged = 0;
         $acceptedChoicesLogged = 0;
+        $orderingDelayMinutes = $this->getOrderingDelayMinutes($fulfillmentMethod->getOrderingDelayMinutes());
 
-        return array_filter($choices, function (TsRangeChoice $choice) use ($cart, &$choicesLogged, &$acceptedChoicesLogged) {
+        $this->logger->info('here');
 
-            $result = $this->shippingDateFilter->accept($cart, $choice->toTsRange());
+        return array_filter($choices, function (TsRangeChoice $choice) use ($cart, $orderingDelayMinutes, &$choicesLogged, &$acceptedChoicesLogged) {
 
-            if ($choicesLogged < self::MAX_CHOICES_LOGGED && $acceptedChoicesLogged < self::MAX_ACCEPTED_CHOICES_LOGGED) {
```

Comment:
that was added to limit the number of choices logged. If we log all possible choices for the next 7 days on each call, we'll get gigabytes of logs on busy instances

Reply:
yes yes I saw

i moved it out to debug but I surely should have not commit that... :sweat_smile: thanks for spotting this out